messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -845,10 +845,14 @@ private void abortAndScheduleRetry(Exception cause) {
             logger.info("Releasing claims and scheduling a new coordination task in {}ms", errorWaitBackOff);
 
             errorWaitBackOff = Math.min(errorWaitBackOff * 2, 60000);
-            abortWorkPackages(cause).thenRun(
-                    () -> {
-                        logger.debug("Work packages have aborted. Scheduling new coordination task to run in {}ms",
-                                     errorWaitBackOff);
+            abortWorkPackages(cause).whenComplete(
+                    (unused, throwable) -> {
+                        if (throwable != null) {
+                            logger.error("An exception occurred during work packages abort on [{}] processor.", name, throwable);
```

Comment:
I would use the "warn" level here since I expect aborting the worker node will be effectively successful, it's just that an exception has occurred in the process. 

To avoid logging in "known" cases, such as the inability to release a token, I would also change the "abortWorkPackage" method by adding an "exceptionally(e -> ...)" to the end of the CompletableFuture chain. Those exceptions can be logged at the info level, since releasing the token on an aborted processor is a best-effort process.

Reply:
Hi @abuijze! Thank you for your review!!
Your suggestions have been implemented!!